Netezos.Tests/Rpc/SettingsFixture.cs

```diff
@@ -1,31 +1,79 @@
-﻿using Dynamic.Json;
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Dynamic.Json;
 using Netezos.Rpc;
-using System;
+using Xunit;
 
 namespace Netezos.Tests.Rpc
 {
-    public class SettingsFixture : IDisposable
+    public class SettingsFixture : IDisposable, IAsyncLifetime
     {
         static readonly object Crit = new object();
 
         public TezosRpc Rpc { get; }
         public string TestContract { get; }
         public string TestDelegate { get; }
         public string TestInactive { get; }
+        private NodeContainer NodeContainer { get; }
+        private int HealthCheckTimeout { get; }
 
         public SettingsFixture()
         {
             lock (Crit)
             {
                 var settings = DJson.Read("../../../Rpc/settings.json");
 
-                Rpc = new TezosRpc(settings.node, 60);
+                NodeContainer = new NodeContainer(settings.sandboxNode.imageName, settings.sandboxNode.tag, settings.sandboxNode.port);
```

Comment:
There should also be a graceful fallback in case docker is not installed on the host system.
Options: 1) skip tests requiring sandbox 2) use public node instead (?)